konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/ListCommand.java

```diff
@@ -120,8 +121,9 @@ private void printServerDetails(int index, String printFormat, String id, String
 
         try {
             configuration = FileUtils.readFileToString(new File(DirectoryFetcher.getServersDataDir(), pid + ".data"), StandardCharsets.UTF_8);
-            InferenceConfiguration inferenceConfiguration = InferenceConfiguration.fromJson(configuration);
-            hostAndPort = String.format("%s:%s", inferenceConfiguration.getHost(), inferenceConfiguration.getPort());
+            JsonNode jsonNode = ObjectMappers.json().readTree(configuration);
+
+            hostAndPort = String.format("%s:%s", jsonNode.get("host"), jsonNode.get("port")).replaceAll("\"", "");
```

Comment:
What was the reasoning for getting it from the JSON instead of the inference configuration?
(I mean I think both should work, just wondering why the change)